source/includes/fact-connect-srv-description.rst

```diff
@@ -0,0 +1,28 @@
+*New in Compass version 1.12*
```

Comment:
We're on version 1.17 already - not sure that this is needed anymore. Thoughts on removing it?